app/routes.py

```diff
@@ -1,2 +1,74 @@
-from flask import Blueprint
+from attr import validate
+from app import db
+from app.models.planet import Planet
 
+from crypt import methods
+from flask import Blueprint , jsonify, abort, make_response, request
+
+
+planet_bp = Blueprint("planets", __name__,url_prefix="/planets")
+
+def validate_planet(cls, planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {planet_id} invalid"}, 400))
+    
+    planet = cls.query.get(planet_id)
+
+    if not planet:
+        abort(make_response({"message":f"{cls.__name__} {planet_id} not found"}, 404))
+    
+    return planet
+
+@planet_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
```

Comment:
Doing some validation on the request body (to make sure it has the required fields) would be appropriate.